tests/test_bgp_neighbor_af.rb

```diff
@@ -150,15 +233,17 @@ def test_props_bool
   def props_bool(af, dbg)
     # These properties have simple boolean states. As such we can use a common
     # set of tests to validate each property.
+    # Common to both XR and nexus
     props = [
       :as_override,
-      :disable_peer_as_check,
       :next_hop_self,
-      :next_hop_third_party,
-      :suppress_inactive,
     ]
 
-    props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
+    if platform == :nexus
+      props << :next_hop_third_party
+      props << :suppress_inactive
+      props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
```

Comment:
Just to be clear:
```
props = [:as_override, :next_hop_self] # XR, not l2vpn/evpn
props = [:as_override, :next_hop_self] # XR, l2vpn/evpn
props = [:as_override, :next_hop_self, :next_hop_third_party, :suppress_inactive] # Nexus, not l2vpn/evpn
props = [:disable_peer_as_check] # Nexus, l2vpn/evpn
```

So it's intentional that Nexus does *not* test the other properties under l2vpn, but *does* test only `:disable_peer_as_check`; and conversely, that XR *does* test the other properties (which Nexus doesn't) under l2vpn but does *not* test `:disable_peer_as_check`?